.travis.yml

```diff
@@ -16,9 +20,13 @@ jobs:
   exclude:
     - os: osx
       env: CMAKE_OPTIONS="-DCMAKE_TOOLCHAIN_FILE=../cmake/Toolchain-cross-mingw32-linux.cmake" # don't test MinGW from OSX
+    - os: linux
+      env: CMAKE_OPTIONS="-DCMAKE_TOOLCHAIN_FILE=../cmake/Toolchain-cross-mingw32-linux.cmake"
+      arch: ppc64le
 
 before_install:
-  - if [ "$TRAVIS_OS_NAME" = "linux" ]; then sudo apt-get update; sudo apt-get --no-install-suggests --no-install-recommends install libsdl2-dev liblzo2-dev gnutls-dev libgcrypt-dev wine; fi
+  - if [ "$TRAVIS_OS_NAME" = "linux" -a "$TRAVIS_ARCH" = "amd64" ]; then sudo apt-get install wine ; fi
```

Comment:
please nest the tests so there is only a single test for "linux". also:
* apt-get update should be called in any case
* apt-get install should be called with the given --options in any case

Reply:
Hi @bk138 , I have nested the tests in a single test for linux as asked.

Thanks !!